App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import Navigator from "./Navigation/NavigationComponent";
+
+export default function App() {
+  return <Navigator style={styles.container} />;
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#fff",
+    alignItems: "center",
+    justifyContent: "center"
+  }
+});
```

Comment:
@bibangamba  Can you create a separate style file for all your styles, so you import them here. It'll makes the code readable